common/djangoapps/course_category/admin.py

```diff
@@ -7,6 +7,32 @@
 from models import CourseCategory, Program
 
 
+class ProductModelMultipleChoiceField(forms.ModelMultipleChoiceField):
+    '''
+    Redefinition of clean method.
+
+    Values that original method recieves by m2m from CourseOverview are course_id (str),
+    while CourseLocator needed.
+    '''
+    def clean(self, value):
+        value = map(CourseKey.from_string, value)
```

Comment:
Please, try to avoid using obsolete `map` and use list comprehensions instead. It makes code more readable, portable and compatible with Python3.

```suggestion
        value = [CourseKey.from_string(v) for v in value]
```

Reply:
Agree